test_util/run-all

```diff
@@ -20,6 +20,14 @@ cleanup_vagrant_leftovers() {
   set -e
 }
 
+replay_logs() {
```

Comment:
Lets simplify this and only do `vagrant ssh $1 -c "'sudo journalctl -n $LOG_REPLAY_LINES'"` further down, removing the function. This is not particularly useful.